cmd/auth/routes.go

```diff
@@ -10,19 +10,26 @@ import (
 	oauthhandler "github.com/skygeario/skygear-server/pkg/auth/handler/oauth"
 	"github.com/skygeario/skygear-server/pkg/auth/handler/session"
 	webapphandler "github.com/skygeario/skygear-server/pkg/auth/handler/webapp"
-	"github.com/skygeario/skygear-server/pkg/core/middleware"
+	"github.com/skygeario/skygear-server/pkg/core/sentry"
 	"github.com/skygeario/skygear-server/pkg/core/server"
 	"github.com/skygeario/skygear-server/pkg/deps"
 	"github.com/skygeario/skygear-server/pkg/middlewares"
 )
 
+func NewRouter(p *deps.RootProvider) *mux.Router {
+	rootRouter := mux.NewRouter()
+	rootRouter.Use(sentry.Middleware(sentry.DefaultClient.Hub))
```

Comment:
Actually sentry need ServerConfig (for constructing request URL), so it need to use `p.Middleware` too. We can do it in another PR.